### PR TITLE
[PPP-11142] Codebase Foundation

### PR DESCRIPTION
**Pull request by jlam-mpc**
Features:
- Added initial directory structure
- Added Layout component with Header and Footer placeholders
- Forecast Filters placeholder
- Homepage Banner placeholder

Update:
- Formatted files with ESLint rules

Fix:
- Added workaround fix for buttons having white text on white background
  - Manual copy of Tailwind preflight without the button background line
  - Added to global.css

### CODE REVIEW

File: src/components/Layout/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                <div className="container m-auto py-1 flex justify-end">
+                    Utility Navigation Menu Items
+                </div>
+            </div>
+            <div className="bg-gray-200">
+                <div className="container m-auto py-5">
+                    U.S. Department of State
+                </div>
+            </div>
+            <div className="container m-auto py-2">
```
Comment (elizabeth-jimenez) [inline src/components/Layout/Header.tsx:21]: We may want to pull this section(L21-34)  into it's own component - my reasoning here is that, on the mockups, it appears to only show on some pages (so not a true header portion).

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
-          <p className="text-2xl text-purple-500">
-            {hello.data ? hello.data.greeting : "Loading tRPC query..."}
-          </p>
-        </div>
-      </main>
-    </>
-  );
+    return (
+        <Layout>
+            <div className='container flex flex-col items-center justify-center gap-12 px-4 py-16'>
```
Comment (elizabeth-jimenez) [inline src/pages/index.tsx:21]: `container` class has a fixed width, do we want this? When I remove it the Public Procurement Project text goes back to being centered

**Comment by elizabeth-jimenez** [comment]
Just a couple comments, but otherwise a great job!

File: src/pages/index.tsx (hunk 2 above)
Comment (jlam-mpc) [reply src/pages/index.tsx:21]: The elements in this div are generated by code at startup. I don't think the homepage is a part of the POC stories we're working on so other than throwing in the layout items I'm not too focused on fixing or adjusting anything on this page yet.

File: src/components/Layout/Header.tsx (hunk 1 above)
Comment (jlam-mpc) [reply src/components/Layout/Header.tsx:21]: I had a discussion with Eli before committing the change to fix this. I believe all of the pages that don't contain this inter-navigation bar are pages directly from state.gov. For now, all of our features will be contain within the forecast tool in which the navigation bar will always exist for. Is it okay to leave it? I did, however, fix the header by adding a prop to denote whether it uses a light or dark background (using temporary tailwind colors, not using the theme yet).

File: src/pages/index.tsx (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/pages/index.tsx:21]: That makes sense :)

File: src/components/Layout/Header.tsx (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/components/Layout/Header.tsx:24]: Ah, that makes sense! Yes, it makes sense to leave it like that - thank you for clarifying that :)

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**